### PR TITLE
NNS1-3435: New icon with color hints

### PR DESCRIPTION
**Pull request by yhabib**
# Motivation

We have a new design for the `VoteConfirmationModal`.

# Changes

-  Update the gix dependency to include new icons.
-  Add a wrapper around the icons in the `VoteConfirmationModal`.

# Tests

[Design reference](https://www.figma.com/design/x8dk5axQUf8Bi43uUsqCHc/NNS-Production-%26-Hand-over?node-id=5486-51881&t=mmR64uy0BVpSlsyi-4)
[Testable version]()

Before:
Screenshot 2024-11-11 at 11 09 28

After:
Screenshot 2024-11-11 at 11 10 07

Before:
Screenshot 2024-11-11 at 11 15 37

After:
Screenshot 2024-11-11 at 11 19 14

Before:
Screenshot 2024-11-11 at 11 15 57

After:
Screenshot 2024-11-11 at 11 18 35

Before:
Screenshot 2024-11-11 at 11 16 15

After:
Screenshot 2024-11-11 at 11 18 44

# Todos

- [ ] Add entry to changelog (if necessary).
Not necessary.

### CODE REVIEW

File: frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte [hunk 1]
```diff
@@ -39,9 +43,30 @@
...
+      border-radius: 50%;
+      aspect-ratio: 1;
+
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+    .yes {
+      color: var(--green);
+      background-color: rgb(from var(--green) r g b / 0.2);
```
Comment (yhabib) [inline frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte:64]: We don't have such a light color. We have two options as far as I can tell:
* apply an opacity filter, though current browser support is not great.
* add color localy, though no support from theme
* add color to GIX library

wdyt?

Comment (dskloetd) [reply frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte:64]: @mstrasinskis does the new scss syntax you were using help with this?

File: frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte [hunk 2]
```diff
@@ -39,9 +43,30 @@
...
+
+      border-radius: 50%;
+      aspect-ratio: 1;
+
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+    .yes {
+      color: var(--green);
```
Comment (dskloetd) [inline frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte:63]: We should have more semantic variables that don't refer to the color directly. What is used in the Figma design?

File: frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte [hunk 3]
```diff
@@ -39,9 +43,30 @@
...
+      height: var(--padding-4x);
+    }
+
+    .icon-wrapper {
       width: var(--padding-6x);
       height: var(--padding-6x);
       margin-bottom: var(--padding);
+
+      border-radius: 50%;
+      aspect-ratio: 1;
```
Comment (dskloetd) [inline frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte:56]: Why is this needed?

Comment (yhabib) [reply frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte]: True, no longer necessary. I took initially another path that required it.

File: frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte (hunk 2 above)
Comment (yhabib) [reply frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte:62]: `Notification/alert` for the red and `Notification/check` for the green.
In https://github.com/dfinity/gix-components/pull/518/files is refer to as `$green-200`

Screenshot 2024-11-11 at 13 54 28

File: frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte [hunk 4]
```diff
@@ -39,9 +43,29 @@
...
+      justify-content: center;
+    }
+    .yes {
+      color: var(--green);
+      background-color: rgb(from var(--green) r g b / 0.25);
+    }
+
+    .no {
+      color: var(--pink);
+      background-color: rgb(from var(--pink) r g b / 0.2);
```
[the remark this answers is not in the document]
Comment (yhabib) [reply frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte]: ohh nice. I didnt see it. thanks

File: frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte (hunk 1 above)
Comment (yhabib) [reply frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte]: Indeed, seems that `--positive-emphasis` and `--negative-emphasis` could be use, though I'd need to add the `--negative-emphasis-light` for the background.

File: frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte (hunk 2 above)
Comment (dskloetd) [reply frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte]: Can you work with @mstrasinskis to make these colors available by their proper name and use those?

**Comment by yhabib** [comment]
test are failing as I removed the test-ids from the icons by accident, I'll revert it in GIX

File: frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte]: You’re absolutely right, let's add `--negative-emphasis-light: var(--pink-200-a25);`!

Comment (yhabib) [reply frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte]: 🫡 https://github.com/dfinity/gix-components/pull/528

Comment (mstrasinskis) [reply frontend/src/lib/modals/proposals/VoteConfirmationModal.svelte]: Thank you.

**Review by mstrasinskis — APPROVED** [review]
Thank you!
